### PR TITLE
#3505: add fuel quantity sensor in telemetry simulator

### PR DESCRIPTION
**Pull request by raphaelcoeffic**
It seemed to be missing. I have still no clue why this sensor is supposed to be sent as 0.01ml. Is there any sensor in this world capable to measure that? Should it be changed as well to something more reasonable? (ml is just enough, IMHO)

### CODE REVIEW

**Comment by projectkk2glider** [comment]
It is helpful if you add text `Closes #3505` to the description of the pull request. This then links the pull request with the original issue.

**Comment by projectkk2glider** [comment]
I would prefer if you put the new sensor below the existing `Fuel` sensor.

File: companion/src/simulation/telemetrysimu.cpp [hunk 1]
```diff
@@ -158,6 +158,7 @@ void TelemetrySimulator::showEvent(QShowEvent *event)
   SET_INSTANCE(t2_inst,    T2_FIRST_ID,            0);
   SET_INSTANCE(rpm_inst,   RPM_FIRST_ID,           DATA_ID_RPM);
   SET_INSTANCE(fuel_inst,  FUEL_FIRST_ID,          0);
+  SET_INSTANCE(fuel_qty_inst, FUEL_QTY_FIRST_ID,   0);
```
Comment (projectkk2glider) [inline companion/src/simulation/telemetrysimu.cpp]: Pleas try to keep the alignment, if needed move the tabs on all other lines.

**Comment by projectkk2glider** [comment]
Closes #3505

**Comment by projectkk2glider** [comment]
Expect for minor issues (comments above) the code works good and will be merged as soon as comments are addressed. Thanks!

**Comment by raphaelcoeffic** [comment]
@projectkk2glider I made the changes, added/updated code from #3507, pushed, but it seems github is rather slow to process new commits sometimes...

**Comment by projectkk2glider** [comment]
Thanks! The code ported to `next` in 06fce5108ecde6fddbdf79ac21fd5f3445951737

**Pull request merged**